src/components/Link/Link.scss

```diff
@@ -8,9 +9,7 @@ $block: '.#{variables.$ns}link';
     -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
     cursor: pointer;
 
-    &:focus {
-        outline: 2px solid var(--g-color-line-misc);
-    }
+    @include mixins.focusable();
```

Comment:
This mixin should also include the next piece of code, to be visible only for keyboard events

Reply:
right, thanks